keras_cv/utils/preprocessing.py

```diff
@@ -323,3 +324,51 @@ def check_fill_mode_and_interpolation(fill_mode, interpolation):
             "Unknown `interpolation` {}. Only `nearest` and "
             "`bilinear` are supported.".format(interpolation)
         )
+
+
+def clip_bounding_box(bounding_boxes, bounding_box_format, images):
```

Comment:
can we handle batching using the _format_inputs and _format_outputs pattern: https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/converters.py#L239

Reply:
Done!